src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,140 +1,121 @@
+import Tippy from '@tippyjs/react';
 import clsx from 'clsx';
-import React, {
-  ReactNode,
-  useState,
-  useRef,
-  useEffect,
-  MutableRefObject,
-} from 'react';
+import * as React from 'react';
 import styles from './Tooltip.module.css';
-import { ESCAPE_KEYCODE, TAB_KEYCODE } from '../../util/keycodes';
-import { Button } from '../Button/Button';
 
-export interface Props {
+// Full list of Tippy props: https://atomiks.github.io/tippyjs/v6/all-props/
+type TooltipProps = {
   /**
-   * Alignment variations
-   * - **right** yields a tooltip that appears to the right of the trigger
-   * - **below** yields a tooltip that appears below the trigger
-   * - **left** yields a tooltip that appears to the left of the trigger
+   * The trigger element the tooltip appears next to.
    */
-  align?: 'right' | 'below' | 'left';
+  children?: React.ReactElement;
   /**
-   * The visually-hidden button text for the tooltip trigger button
+   * The trigger element the tooltip appears next to.
+   *
+   * Use this instead of `children` if the trigger element is being
+   * stored in a ref. Most cases will use `children` and not
+   * `reference`.
    */
-  buttonText?: string;
+  reference?: React.RefObject<Element> | Element;
   /**
-   * Child node(s) that can be nested inside component. This gets displayed inside a `TextPassage` inside the tooltip
+   * The content of the tooltip bubble.
    */
-  children?: ReactNode;
+  content?: React.ReactNode;
```

Comment:
I wonder if this should be `text` based on the prop guidelines Big Medium uses.

Reply:
Changed to `text`, thanks